packages/web-sdk/src/transports/otlp/payload/transform/toResourceLog.test.ts

```diff
@@ -19,6 +19,11 @@ const item: Readonly<TransportItem<LogEvent>> = {
       mobile: false,
       userAgent: 'browser-ua-string',
       language: 'browser-language',
+      brands: [
+        { brand: 'Google Chrome', version: '111' },
+        { brand: 'Not(A:Brand', version: '8' },
```

Comment:
Is this testcase, just to illustrate the input is not sanitised? By the way, thanks for testing even minor changes like this